tests/plugins/templates/manual/insertTemplate.md

```diff
@@ -0,0 +1,27 @@
+@bender-tags: 4.16.2, feature
```

Comment:
```suggestion
@bender-tags: 4.17.0, feature, 4850
```

It will be released in 4.17.0. Please update version tag accordingly and add issue reference.